pkg/storage/postgresql/findings.go

```diff
@@ -658,9 +684,11 @@ func buildFindingExpandedAtTime(row map[string]interface{}, atTime time.Time) (m
 	}
 	expiredAt := parseStringArray(expired)
 
-	err = fillFindingExposures(&f, foundAt, fixedAt, expiredAt, atTime)
-	if err != nil {
-		return model.FindingExpanded{}, err
+	if f.Status != statusFalsePositive {
```

Comment:
I would maybe add a comment here so it's clear why we do this.

Reply:
Agree! Done!